### DISCUSSION TITLE
No AMD CBS or PBS options

### DISCUSSION THREAD

**Discussion by reno911unlocred** [DavidS95/Smokeless_UMAF#83] [Q&A]
Please help. I am rying to use this as per the instructions on **[this](https://www.youtube.com/watch?v=LNEI7BTc87Q&t=1s)** video and when i get to the EFI menu and click con either option of the device manager, nothing comes up, both only have this simple "Driver health manager" entry and nothing more. What am i missing? do i have to enable or disable something in the main bios of the deck or something beforehand?
![WhatsApp Image 2024-11-07 at 15 46 31_d6dffaf5](https://github.com/user-attachments/assets/12dc4284-7443-4db1-b735-63d74118e955)
![WhatsApp Image 2024-11-07 at 15 46 31_58bddc49](https://github.com/user-attachments/assets/1144aac9-70b0-47ec-a5de-442a17961aaa)
![WhatsApp Image 2024-11-07 at 15 46 31_86157501](https://github.com/user-attachments/assets/714873a9-f163-4bef-a35c-64b0eb5e2474)
![WhatsApp Image 2024-11-07 at 15 46 30_cb6cd8db](https://github.com/user-attachments/assets/2917a741-2d2a-43dc-96a8-cf5a879ae83a)

**Comment by Malumen** [comment]
I've seen this in some other googling. You're not done yet. Please redo. You also can use a keyboard. Use other video guides.

**Reply by reno911unlocred** [reply]
What do you mena redo? I have re tried many times changing many settings in bios and nothing worked. I ended up selling my steamdeck in favour of a legion go, and on the legion this wors so i think i wasn't doing something wrong, it is fairly simple, disable secure boot and boot from the usb... I have done it on the legion so.. it was something else and i guess we wil never know or anyone else with the same issue. And yes, i was obviously using a keyboard, otherwise i would not have been able to move in the options. Other videos guides would have told me to do the same thing or things i have already tried.

**Reply by DGWumpscuT** [reply]
I have the same Problem what do i have to do?  Steamdeck 64gb

**Reply by wiebeteselle-lab** [reply]
Hey so i also had this issue turns out after the oled came out they removed the overclocking and tpd and all that stuff there is no way to unlock it unless you are trying to flash a costum or older bios
